xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -1189,7 +1189,8 @@ bool CDVDDemuxFFmpeg::SeekTime(double time, bool backwards, double* startpts)
     }
 
     AVStream* st = m_pFormatContext->streams[m_seekStream];
-    seek_pts = av_rescale(m_startTime + time / 1000, st->time_base.den, st->time_base.num);
+    seek_pts =
+      av_rescale(static_cast<int64_t>(m_startTime + time / 1000), st->time_base.den, st->time_base.num);
```

Comment:
> The `ColumnLimit` in `.clang-format` is set to `100` which defines line length [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#3-formatting

Reply:
I got the .clang-format file to work for me finally and reformatted all the affected areas.  I don't think you'll like that it put 4 spaces instead of 2 in one place, but that's what it did so I left it alone